backend/src/main/java/capstone/backend/services/ProductService.java

```diff
@@ -0,0 +1,59 @@
+package capstone.backend.services;
+
+
+import capstone.backend.mapper.ProductMapper;
+import capstone.backend.model.dto.ProductDTO;
+import capstone.backend.model.exception.ProductIdAlreadyTakenException;
+import capstone.backend.model.exception.ProductNotFoundException;
+import capstone.backend.repo.ProductRepo;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductService {
+
+    private final ProductRepo repo;
```

Comment:
As long as I anyway just use one repository in a class I'm happy with this simplified naming. Since this line is at the top  of the class, the reader knows immediately which repo is used in the class and it keeps the line lengths shorter.